requirements/requirements_asr.txt

```diff
@@ -5,7 +5,7 @@ ipywidgets
 jiwer
 kaldi-python-io
 kaldiio
-lhotse>=1.19.2
+lhotse==1.19.2
```

Comment:
Is it necessary to pin main ? What about r1.23 - that needs the pin too

Reply:
Yes it is necessary